ci/container-canary/notebooks.yml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
```

Comment:
Same here, it's perhaps not clear that this is a test.

Reply:
resolved in https://github.com/rapidsai/docker/pull/670#discussion_r2021585598